web/tests/narrow_state.test.js

```diff
@@ -59,16 +59,17 @@ test("stream", () => {
     assert.ok(narrow_state.is_for_stream_id(test_stream.stream_id));
 
     const expected_terms = [
-        {negated: false, operator: "channel", operand: "Test"},
+        {negated: false, operator: "channel", operand: "15"},
         {negated: false, operator: "topic", operand: "Bar"},
         {negated: false, operator: "search", operand: "yo"},
     ];
 
     const public_terms = narrow_state.public_search_terms();
     assert.deepEqual(public_terms, expected_terms);
-    assert.equal(narrow_state.search_string(), "channel:Test topic:Bar yo");
+    assert.equal(narrow_state.search_string(), "channel:15 topic:Bar yo");
```

Comment:
Is `narrow_state.search_string` a function we should be garbage-collecting at this point? I'm not sure it has any callers.

Reply:
Great find! Yeah looks like no callers. I'll add a prep commit removing that